cypress/integration/executions_spec.js

```diff
@@ -69,5 +69,42 @@ describe('Dashboard Executions Page', () => {
         });
       });
     });
+
+    it('should show a single execution', () => {
+      cy.contains('nav li a', 'Executions').as('executions');
+      cy.get('@executions').should('have.attr', 'href', '#/executions');
+      cy.get('@executions').click();
+
+      cy.url().should('include', 'executions');
+      cy.contains('.heading--xlarge', 'Executions');
+      const executionName = '50eaad71-bba8-4376-83d7-bb9cc1309b92';
+      const executionArn = 'arn:aws:states:us-east-1:596205514787:execution:TestSourceIntegrationIngestGranuleStateMachine-MOyI0myKEXzf:50eaad71-bba8-4376-83d7-bb9cc1309b92';
+      const stateMachine = 'arn:aws:states:us-east-1:596205514787:stateMachine:TestSourceIntegrationIngestGranuleStateMachine-MOyI0myKEXzf';
+      cy.get('table tbody tr td[class=table__main-asset]').within(() => {
+        cy.get(`a[title=${executionName}]`).click({force: true});
+      });
+
+      cy.contains('.heading--large', 'Execution');
+      cy.contains('.heading--medium', 'Visual workflow');
+
+      cy.get('.table--wrapper')
+        .within(() => {
+          cy.get('tr[data-value=31]').children('td').as('items');
+          cy.get('@items').eq(0).should('have.text', '31');
+          cy.get('@items').eq(1).should('have.text', 'ExecutionSucceeded');
+          cy.get('@items').eq(2).invoke('text').should('match', /\d{2}:\d{2}:\d{2} \d{2}\/\d{2}\/\d{2}$/);
```

Comment:
Can we test the timestamp matches the one from fixture, even if they are in different format?

Reply:
Okay using the real fixture times in StartDate and stopDate. Didn't change this one because it's now in a loop with different times